src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Folding/RazorFoldingRangeRequestParam.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
```

Comment:
nit: Few extra lines

Reply:
The bane of my existence is extra blank lines...